parent/pom.xml

```diff
@@ -205,7 +205,7 @@
         <geronimo-ws-metadata-spec-version>1.1.3</geronimo-ws-metadata-spec-version>
         <gmetric4j-version>1.0.10</gmetric4j-version>
         <google-guava-version>19.0</google-guava-version>
-        <google-pubsub-guava-version>28.2-jre</google-pubsub-guava-version>
+        <google-pubsub-guava-version>29.0-jre</google-pubsub-guava-version>
```

Comment:
I think this require a check on the karaf feature if needed

Reply:
You mean in the camel-dependencies project right? I've updated it there now.

Reply:
We need to check the camel-googke-pubsub karaf feature in the camel-karaf repo, but we can do that later